packages/tradershub/src/features/cfd/screens/Jurisdiction/JurisdictionScreen.tsx

```diff
@@ -48,11 +48,13 @@ const JurisdictionScreen = ({
     return (
         <div
             className={qtMerge(
-                'flex flex-col h-auto w-[85vw] items-center justify-center my-auto mx-1500 sm:h-[75vh] transition-all ease-in duration-[0.6s]',
+                `flex flex-col ${
+                    marketType === MarketType.FINANCIAL ? 'w-[1200px] h-[700px]' : 'w-[1040px] h-[700px]'
+                }  items-center justify-center transition-all ease-in duration-[0.6s]`,
```

Comment:
h-[700px] is same for both condition maybe it should be in main class
```suggestion
                    marketType === MarketType.FINANCIAL ? 'w-[1200px]' : 'w-[1040px]'
                }  items-center justify-center transition-all ease-in duration-[0.6s] h-[700px]`,
```